### PR TITLE
Save city/state in zip3 table for rate engine

### PR DESCRIPTION
**Pull request by reggieriser**
## Description

The current database design for the GHC rate engine associates a city/state name with a service area.  It turns out, though, that the pricing template spreadsheet has multiple cities noted per service area (which may span multiple zip3s).  We previously were just picking one of those cities and assigning it to the service area, thus dropping some city information.  This PR moves the city/state from the `re_domestic_service_areas` table to the `re_zip3s` table so we no longer have to discard that data.  Each zip3 for a service area can now have a distinct city/state.

## Reviewer Notes

Because the output of our `ghc-pricing-parser` command had gotten noisy, I cleaned up some logs that we put in when first working with this that aren't particularly useful anymore.  I also did some basic normalizing of some logging output.  There's [another story](https://dp3.atlassian.net/browse/MB-1301) to go deeper into this logging cleanup.

## Setup

To do the import:
- `rm bin/ghc-pricing-parser`
- `make bin/ghc-pricing-parser`
- `make db_dev_reset db_dev_migrate`
- `ghc-pricing-parser --filename pkg/parser/pricing/fixtures/pricing_template_2019-09-19_fake-data.xlsx --contract-code TEST`

After running this, you should see city/state populated in the `re_zip3s` table and no longer on the `re_domestic_service_areas` table.

To run the tests:
- `make server_test`

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] The requirements listed in [Querying the Database Safely(https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1313) for this change

### CODE REVIEW

File: migrations/app/schema/20200217175555_move-re-city-state.up.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+-- Make the columns not nullable
+ALTER TABLE re_zip3s
+    ALTER COLUMN base_point_city SET NOT NULL,
+    ALTER COLUMN state SET NOT NULL;
+
+-- Drop the (now) old city/state columns from the re_domestic_service_areas table
+ALTER TABLE re_domestic_service_areas
+    DROP COLUMN base_point_city,
+    DROP COLUMN state;
```
Comment (chrisgilmerproj) [inline migrations/app/schema/20200217175555_move-re-city-state.up.sql]: I don't think you can drop the columns until you've also removed the associated code in a deploy. So you may have to do this in two steps.

Comment (reggieriser) [reply migrations/app/schema/20200217175555_move-re-city-state.up.sql]: There's no data in these tables nor any reachable code that actively uses them in a deployed environment yet.

File: pkg/services/ghcimport/ghc_rateengine_importer.go [hunk 2]
```diff
@@ -22,72 +22,86 @@ type GHCRateEngineImporter struct {
 
 func (gre *GHCRateEngineImporter) runImports(dbTx *pop.Connection) error {
 	// Reference tables
+	gre.Logger.Info("Importing contract")
```
Comment (jacquelineIO) [inline pkg/services/ghcimport/ghc_rateengine_importer.go]: thanks for adding these

File: pkg/services/ghcimport/import_re_domestic_service_area.go [hunk 3]
```diff
@@ -28,9 +36,6 @@ func (gre *GHCRateEngineImporter) importREDomesticServiceArea(dbTx *pop.Connecti
 		return err
 	}
 
-	// TODO: We aren't currently cleaning up any records that may not be part of the latest
```
Comment (jacquelineIO) [inline pkg/services/ghcimport/import_re_domestic_service_area.go]: yay!

**Pull request merged**